client/cmd.go

```diff
@@ -2,11 +2,11 @@ package client
 
 import (
 	"crypto/tls"
+	"errors"
```

Comment:
Yes, I've checked the two cases where I used the stdlib instead of cockroachdb/errors.
Here I don't think it's useful, as it's about command validation. I will still use `cockroachdb/errors` just in case, so there won't be any behavior change in this PR.